features/exploredash/src/main/java/org/dash/wallet/features/exploredash/ui/dialogs/FiltersDialog.kt

```diff
@@ -186,7 +186,7 @@ class FiltersDialog: OffsetDialogFragment<ConstraintLayout>() {
         binding.radiusLabel.isVisible = true
         binding.radiusCard.isVisible = true
 
-        selectedRadiusOption = viewModel.selectedRadiusOption
+        selectedRadiusOption = viewModel.selectedRadiusOption.value!!
```

Comment:
See the comment above. Here, we should use default radius value if `selectedRadiusOption` is null.